ic-helpers/src/macros.rs

```diff
@@ -0,0 +1,49 @@
+#[macro_export]
```

Comment:
Instead of all this it's better to just use `ic-kit` crate. It does the same thing, but better.